supercollider/supercollider-chocolatey.nuspec

```diff
@@ -25,7 +25,7 @@
     Requires the git.install package to be installed so that the SuperCollider Quarks feature
     works correctly.</description>
   <dependencies>
-    <dependency id="git" version="2.5" />
+    <dependency id="git" version="2.33.1" />
```

Comment:
I don't remember how chocolatey works...  will this _require_ version 2.33.1 or will it lower the minimum required version to 2.33.1?

Reply:
At the time of the PR, the most recent version of the git choco package was `2.33.1`. It appears to be `2.34.1` now.